### ISSUE TITLE
Wrong sort order

### ISSUE THREAD

**Issue by norbertgaal** [nightscout/cgm-remote-monitor#3666]
In the dev branch the sort order has changed:

This is the output of /api/v1/entries/sgv.json?count=6 using the **master** branch:

```
[
    {
        "_id": "5b3d289c14265f09a6775253",
        "date": 1530734744140,
        "dateString": "2018-07-04T21:05:44.140+0100",
        "delta": 5,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3383,
        "noise": 1,
        "rssi": 100,
        "sgv": 133,
        "sysTime": "2018-07-04T21:05:44.140+0100",
        "type": "sgv",
        "unfiltered": 3663
    },
    {
        "_id": "5b3d277014265f09a6773109",
        "date": 1530734444219,
        "dateString": "2018-07-04T21:00:44.219+0100",
        "delta": 4,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3253,
        "noise": 1,
        "rssi": 100,
        "sgv": 126,
        "sysTime": "2018-07-04T21:00:44.219+0100",
        "type": "sgv",
        "unfiltered": 3473
    },
    {
        "_id": "5b3d264814265f09a6770ffb",
        "date": 1530734148285,
        "dateString": "2018-07-04T20:55:48.285+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3143,
        "noise": 1,
        "rssi": 100,
        "sgv": 119,
        "sysTime": "2018-07-04T20:55:48.285+0100",
        "type": "sgv",
        "unfiltered": 3278
    },
    {
        "_id": "5b3d251c14265f09a676eeaa",
        "date": 1530733847925,
        "dateString": "2018-07-04T20:50:47.925+0100",
        "delta": 3.5,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3041,
        "noise": 1,
        "rssi": 100,
        "sgv": 117,
        "sysTime": "2018-07-04T20:50:47.925+0100",
        "type": "sgv",
        "unfiltered": 3213
    },
    {
        "_id": "5b3d23ec14265f09a676cc84",
        "date": 1530733544239,
        "dateString": "2018-07-04T20:45:44.239+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 2950,
        "noise": 1,
        "rssi": 100,
        "sgv": 113,
        "sysTime": "2018-07-04T20:45:44.239+0100",
        "type": "sgv",
        "unfiltered": 3105
    },
    {
        "_id": "5b3d22c114265f09a676aafa",
        "date": 1530733244341,
        "dateString": "2018-07-04T20:40:44.341+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 2869,
        "noise": 1,
        "rssi": 100,
        "sgv": 110,
        "sysTime": "2018-07-04T20:40:44.341+0100",
        "type": "sgv",
        "unfiltered": 3007
    }
]
```

And this is the output of the same command using the current **dev** branch. As you can see the order of the items have changed. All of the env variables and the mongodb behind it is the same:

```
[
    {
        "_id": "5b3d22c114265f09a676aafa",
        "date": 1530733244341,
        "dateString": "2018-07-04T20:40:44.341+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 2869,
        "noise": 1,
        "rssi": 100,
        "sgv": 110,
        "sysTime": "2018-07-04T20:40:44.341+0100",
        "type": "sgv",
        "unfiltered": 3007
    },
    {
        "_id": "5b3d23ec14265f09a676cc84",
        "date": 1530733544239,
        "dateString": "2018-07-04T20:45:44.239+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 2950,
        "noise": 1,
        "rssi": 100,
        "sgv": 113,
        "sysTime": "2018-07-04T20:45:44.239+0100",
        "type": "sgv",
        "unfiltered": 3105
    },
    {
        "_id": "5b3d251c14265f09a676eeaa",
        "date": 1530733847925,
        "dateString": "2018-07-04T20:50:47.925+0100",
        "delta": 3.5,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3041,
        "noise": 1,
        "rssi": 100,
        "sgv": 117,
        "sysTime": "2018-07-04T20:50:47.925+0100",
        "type": "sgv",
        "unfiltered": 3213
    },
    {
        "_id": "5b3d264814265f09a6770ffb",
        "date": 1530734148285,
        "dateString": "2018-07-04T20:55:48.285+0100",
        "delta": 3,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3143,
        "noise": 1,
        "rssi": 100,
        "sgv": 119,
        "sysTime": "2018-07-04T20:55:48.285+0100",
        "type": "sgv",
        "unfiltered": 3278
    },
    {
        "_id": "5b3d277014265f09a6773109",
        "date": 1530734444219,
        "dateString": "2018-07-04T21:00:44.219+0100",
        "delta": 4,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3253,
        "noise": 1,
        "rssi": 100,
        "sgv": 126,
        "sysTime": "2018-07-04T21:00:44.219+0100",
        "type": "sgv",
        "unfiltered": 3473
    },
    {
        "_id": "5b3d289c14265f09a6775253",
        "date": 1530734744140,
        "dateString": "2018-07-04T21:05:44.140+0100",
        "delta": 5,
        "device": "xDrip-DexcomG5 G5 Native",
        "direction": "Flat",
        "filtered": 3383,
        "noise": 1,
        "rssi": 100,
        "sgv": 133,
        "sysTime": "2018-07-04T21:05:44.140+0100",
        "type": "sgv",
        "unfiltered": 3663
    }
]

```

Link to the Gitter conversation with @PieterGit [https://gitter.im/nightscout/public?at=5b3d4ecca99e1e52b716bde7](https://gitter.im/nightscout/public?at=5b3d4ecca99e1e52b716bde7)

**Comment by PieterGit** [comment]
Summary of Gitter chat:
- tested with current dev branch (up and including https://github.com/nightscout/cgm-remote-monitor/pull/3661 )
- master works, so it's a regression
- reverting to Jun 9, https://github.com/PieterGit/cgm-remote-monitor/tree/201807_revert3 did not fix the problem

**Comment by sulkaharo** [comment]
TESTING: bfce345 from April 3 exhibits the issue, 93968f3 doesn't

**Comment by PieterGit** [comment]
Started upgrading sgvdata. Node 8 almost seems ready (1 failing MQTT test), but Node 10 has 3 failing tests all to do with sgv data. See https://github.com/bewest/sgvdata/pull/3 and https://github.com/nightscout/cgm-remote-monitor/pull/3671

Help wanted to solve these nasty issues...

**Comment by sulkaharo** [comment]
This diff should contain the answer. I have the exact same code running, where when this diff is applied to npm-shrinkwrap.json, the sorting breaks
```
diff -r tmpdiff1/npm-shrinkwrap.json tmpdiff2/npm-shrinkwrap.json
464,510d463
<     "broadway": {
<       "version": "0.3.6",
<       "resolved": "https://registry.npmjs.org/broadway/-/broadway-0.3.6.tgz",
<       "integrity": "sha1-fb7waLlUt5B5Jf1USWO1eKkCuno=",
<       "requires": {
<         "cliff": "0.1.9",
<         "eventemitter2": "0.4.14",
<         "nconf": "0.6.9",
<         "utile": "0.2.1",
<         "winston": "0.8.0"
<       },
<       "dependencies": {
<         "async": {
<           "version": "0.2.10",
<           "resolved": "https://registry.npmjs.org/async/-/async-0.2.10.tgz",
<           "integrity": "sha1-trvgsGdLnXGXCMo43owjfLUmw9E="
<         },
<         "cliff": {
<           "version": "0.1.9",
<           "resolved": "https://registry.npmjs.org/cliff/-/cliff-0.1.9.tgz",
<           "integrity": "sha1-ohHgnGo947oa8n0EnTASUNGIErw=",
<           "requires": {
<             "colors": "0.6.2",
<             "eyes": "0.1.8",
<             "winston": "0.8.0"
<           }
<         },
<         "colors": {
<           "version": "0.6.2",
<           "resolved": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz",
<           "integrity": "sha1-JCP+ZnisDF2uiFLl0OW+CMmXq8w="
<         },
<         "winston": {
<           "version": "0.8.0",
<           "resolved": "https://registry.npmjs.org/winston/-/winston-0.8.0.tgz",
<           "integrity": "sha1-YdCDD6aZcGISIGsKK1ymmpMENmg=",
<           "requires": {
<             "async": "0.2.10",
<             "colors": "0.6.2",
<             "cycle": "1.0.3",
<             "eyes": "0.1.8",
<             "pkginfo": "0.3.1",
<             "stack-trace": "0.0.10"
<           }
<         }
<       }
<     },
656,663d608
<     "caller": {
<       "version": "0.0.1",
<       "resolved": "https://registry.npmjs.org/caller/-/caller-0.0.1.tgz",
<       "integrity": "sha1-83odbqEOgp2UchrimpC7T7Uqt2c=",
<       "requires": {
<         "tape": "2.3.3"
<       }
<     },
798,814d742
<     "cliff": {
<       "version": "0.1.10",
<       "resolved": "https://registry.npmjs.org/cliff/-/cliff-0.1.10.tgz",
<       "integrity": "sha1-U74z6p9ZvshWCe4wCsQgdgPlIBM=",
<       "requires": {
<         "colors": "1.0.3",
<         "eyes": "0.1.8",
<         "winston": "0.8.3"
<       },
<       "dependencies": {
<         "colors": {
<           "version": "1.0.3",
<           "resolved": "https://registry.npmjs.org/colors/-/colors-1.0.3.tgz",
<           "integrity": "sha1-BDP0TYCWgP3rYO0mDxsMJi6CpAs="
<         }
<       }
<     },
1243,1247d1170
<     "cycle": {
<       "version": "1.0.3",
<       "resolved": "https://registry.npmjs.org/cycle/-/cycle-1.0.3.tgz",
<       "integrity": "sha1-IegLK+hYD5i0aPN5QwZisEbDStI="
<     },
1294,1298d1216
<     "deep-equal": {
<       "version": "1.0.1",
<       "resolved": "https://registry.npmjs.org/deep-equal/-/deep-equal-1.0.1.tgz",
<       "integrity": "sha1-9dJgKStmDghO/0zbyfCK0yR0SLU="
<     },
1368,1372d1285
<     "director": {
<       "version": "1.2.7",
<       "resolved": "https://registry.npmjs.org/director/-/director-1.2.7.tgz",
<       "integrity": "sha1-v9N0EHX9f7GlsuE2WMX0vsd3NvM="
<     },
1733,1737d1645
<     "eventemitter2": {
<       "version": "0.4.14",
<       "resolved": "https://registry.npmjs.org/eventemitter2/-/eventemitter2-0.4.14.tgz",
<       "integrity": "sha1-j2G3XN4BKy6esoTUVFWDtWQ7Yas="
<     },
1937,1941d1844
<     "eyes": {
<       "version": "0.1.8",
<       "resolved": "https://registry.npmjs.org/eyes/-/eyes-0.1.8.tgz",
<       "integrity": "sha1-Ys8SAjTGg3hdkCNIqADvPgzCC8A="
<     },
2057,2078d1959
<     "flatiron": {
<       "version": "0.4.3",
<       "resolved": "https://registry.npmjs.org/flatiron/-/flatiron-0.4.3.tgz",
<       "integrity": "sha1-JIz3mj2n19w3nioRySonGcu1QPY=",
<       "requires": {
<         "broadway": "0.3.6",
<         "director": "1.2.7",
<         "optimist": "0.6.0",
<         "prompt": "0.2.14"
<       },
<       "dependencies": {
<         "optimist": {
<           "version": "0.6.0",
<           "resolved": "https://registry.npmjs.org/optimist/-/optimist-0.6.0.tgz",
<           "integrity": "sha1-aUJIJvNAX3nxQub8PZrljU27kgA=",
<           "requires": {
<             "minimist": "0.0.10",
<             "wordwrap": "0.0.3"
<           }
<         }
<       }
<     },
2102,2135d1982
<     "forever": {
<       "version": "0.15.3",
<       "resolved": "https://registry.npmjs.org/forever/-/forever-0.15.3.tgz",
<       "integrity": "sha1-d9nX4V/S9RGtnYShEMfdj8js68I=",
<       "requires": {
<         "cliff": "0.1.10",
<         "clone": "1.0.3",
<         "colors": "0.6.2",
<         "flatiron": "0.4.3",
<         "forever-monitor": "1.7.1",
<         "nconf": "0.6.9",
<         "nssocket": "0.5.3",
<         "object-assign": "3.0.0",
<         "optimist": "0.6.1",
<         "path-is-absolute": "1.0.1",
<         "prettyjson": "1.2.1",
<         "shush": "1.0.0",
<         "timespan": "2.3.0",
<         "utile": "0.2.1",
<         "winston": "0.8.3"
<       },
<       "dependencies": {
<         "colors": {
<           "version": "0.6.2",
<           "resolved": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz",
<           "integrity": "sha1-JCP+ZnisDF2uiFLl0OW+CMmXq8w="
<         },
<         "object-assign": {
<           "version": "3.0.0",
<           "resolved": "https://registry.npmjs.org/object-assign/-/object-assign-3.0.0.tgz",
<           "integrity": "sha1-m+3VygiXlJvKR+f/QIBi1Un1h/I="
<         }
<       }
<     },
2141,2152d1987
<     "forever-monitor": {
<       "version": "1.7.1",
<       "resolved": "https://registry.npmjs.org/forever-monitor/-/forever-monitor-1.7.1.tgz",
<       "integrity": "sha1-XYIPSjp42y2BriZx8Vi56GoJG7g=",
<       "requires": {
<         "broadway": "0.3.6",
<         "chokidar": "1.7.0",
<         "minimatch": "3.0.4",
<         "ps-tree": "0.0.3",
<         "utile": "0.2.1"
<       }
<     },
2709,2713d2543
<     "i": {
<       "version": "0.3.6",
<       "resolved": "https://registry.npmjs.org/i/-/i-0.3.6.tgz",
<       "integrity": "sha1-2WyScyB28HJxG2sQ/X1PZa2O4j0="
<     },
2822c2652,2653
<       "integrity": "sha1-BTfLedr1m1mhpRff9wbIbsA5Fi4="
---
>       "integrity": "sha1-BTfLedr1m1mhpRff9wbIbsA5Fi4=",
>       "optional": true
3278,3282d3108
<     "lazy": {
<       "version": "1.0.11",
<       "resolved": "https://registry.npmjs.org/lazy/-/lazy-1.0.11.tgz",
<       "integrity": "sha1-2qBoIGKCVCwIgojpdcKXwa53tpA="
<     },
3666c3492,3493
<       "integrity": "sha1-3j+YVD2/lggr5IrRoMfNqDYwHc8="
---
>       "integrity": "sha1-3j+YVD2/lggr5IrRoMfNqDYwHc8=",
>       "dev": true
3825,3829d3651
<     "mute-stream": {
<       "version": "0.0.7",
<       "resolved": "https://registry.npmjs.org/mute-stream/-/mute-stream-0.0.7.tgz",
<       "integrity": "sha1-MHXOk7whuPq0PhvE2n6BFe0ee6s="
<     },
3836,3866d3657
<     "nconf": {
<       "version": "0.6.9",
<       "resolved": "https://registry.npmjs.org/nconf/-/nconf-0.6.9.tgz",
<       "integrity": "sha1-lXDvFe1vmuays8jV5xtm0xk81mE=",
<       "requires": {
<         "async": "0.2.9",
<         "ini": "1.3.4",
<         "optimist": "0.6.0"
<       },
<       "dependencies": {
<         "async": {
<           "version": "0.2.9",
<           "resolved": "https://registry.npmjs.org/async/-/async-0.2.9.tgz",
<           "integrity": "sha1-32MGD789Myhqdqr21Vophtn/hhk="
<         },
<         "optimist": {
<           "version": "0.6.0",
<           "resolved": "https://registry.npmjs.org/optimist/-/optimist-0.6.0.tgz",
<           "integrity": "sha1-aUJIJvNAX3nxQub8PZrljU27kgA=",
<           "requires": {
<             "minimist": "0.0.10",
<             "wordwrap": "0.0.3"
<           }
<         }
<       }
<     },
<     "ncp": {
<       "version": "0.4.2",
<       "resolved": "https://registry.npmjs.org/ncp/-/ncp-0.4.2.tgz",
<       "integrity": "sha1-q8xsvT7C7Spyn/bnwfqPAXhKhXQ="
<     },
4129,4137d3919
<     "nssocket": {
<       "version": "0.5.3",
<       "resolved": "https://registry.npmjs.org/nssocket/-/nssocket-0.5.3.tgz",
<       "integrity": "sha1-iDyi7GBfXtZKTVGQsmJUAZKPj40=",
<       "requires": {
<         "eventemitter2": "0.4.14",
<         "lazy": "1.0.11"
<       }
<     },
4208a3991
>       "dev": true,
4440,4444d4222
<     "pkginfo": {
<       "version": "0.3.1",
<       "resolved": "https://registry.npmjs.org/pkginfo/-/pkginfo-0.3.1.tgz",
<       "integrity": "sha1-Wyn2qB9wcXFC4J52W76rl7T4HiE="
<     },
5037,5048d4814
<     "prompt": {
<       "version": "0.2.14",
<       "resolved": "https://registry.npmjs.org/prompt/-/prompt-0.2.14.tgz",
<       "integrity": "sha1-V3VPZPVD/XsIRXB8gY7OYY8F/9w=",
<       "requires": {
<         "pkginfo": "0.3.1",
<         "read": "1.0.7",
<         "revalidator": "0.1.8",
<         "utile": "0.2.1",
<         "winston": "0.8.3"
<       }
<     },
5072,5097d4837
<     "ps-tree": {
<       "version": "0.0.3",
<       "resolved": "https://registry.npmjs.org/ps-tree/-/ps-tree-0.0.3.tgz",
<       "integrity": "sha1-2/jXUqf+Ivp9WGNWiUmWEOknbdw=",
<       "requires": {
<         "event-stream": "0.5.3"
<       },
<       "dependencies": {
<         "event-stream": {
<           "version": "0.5.3",
<           "resolved": "https://registry.npmjs.org/event-stream/-/event-stream-0.5.3.tgz",
<           "integrity": "sha1-t3uTCfcQet3+q2PwwOr9jbC9jBw=",
<           "requires": {
<             "optimist": "0.2.8"
<           }
<         },
<         "optimist": {
<           "version": "0.2.8",
<           "resolved": "https://registry.npmjs.org/optimist/-/optimist-0.2.8.tgz",
<           "integrity": "sha1-6YGrfiaLRXlIWTtVZ0wJmoFcrDE=",
<           "requires": {
<             "wordwrap": "0.0.3"
<           }
<         }
<       }
<     },
5249,5256d4988
<     "read": {
<       "version": "1.0.7",
<       "resolved": "https://registry.npmjs.org/read/-/read-1.0.7.tgz",
<       "integrity": "sha1-s9oZvQUkMal2cdRKQmNK33ELQMQ=",
<       "requires": {
<         "mute-stream": "0.0.7"
<       }
<     },
5594,5606d5325
<     "resumer": {
<       "version": "0.0.0",
<       "resolved": "https://registry.npmjs.org/resumer/-/resumer-0.0.0.tgz",
<       "integrity": "sha1-8ej0YeQGS6Oegq883CqMiT0HZ1k=",
<       "requires": {
<         "through": "2.3.8"
<       }
<     },
<     "revalidator": {
<       "version": "0.1.8",
<       "resolved": "https://registry.npmjs.org/revalidator/-/revalidator-0.1.8.tgz",
<       "integrity": "sha1-/s5hv6DBtSoga9axgZgYS91SOjs="
<     },
5742c5461
<           "resolved": "http://registry.npmjs.org/event-stream/-/event-stream-3.1.7.tgz",
---
>           "resolved": "https://registry.npmjs.org/event-stream/-/event-stream-3.1.7.tgz",
5983,5998d5701
<     "shush": {
<       "version": "1.0.0",
<       "resolved": "https://registry.npmjs.org/shush/-/shush-1.0.0.tgz",
<       "integrity": "sha1-wnQVqeRY8v7TmyfPjrN8ADeCtDE=",
<       "requires": {
<         "caller": "0.0.1",
<         "strip-json-comments": "0.1.3"
<       },
<       "dependencies": {
<         "strip-json-comments": {
<           "version": "0.1.3",
<           "resolved": "https://registry.npmjs.org/strip-json-comments/-/strip-json-comments-0.1.3.tgz",
<           "integrity": "sha1-Fkxk43Coo8wAyeAbU55WmCPw7lQ="
<         }
<       }
<     },
6220,6224d5922
<     "stack-trace": {
<       "version": "0.0.10",
<       "resolved": "https://registry.npmjs.org/stack-trace/-/stack-trace-0.0.10.tgz",
<       "integrity": "sha1-VHxws0fo0ytOEI6hoqFZ5f3eGcA="
<     },
6439,6463d6136
<     "tape": {
<       "version": "2.3.3",
<       "resolved": "https://registry.npmjs.org/tape/-/tape-2.3.3.tgz",
<       "integrity": "sha1-Lnzgox3wn41oUWZKcYQuDKUFevc=",
<       "requires": {
<         "deep-equal": "0.1.2",
<         "defined": "0.0.0",
<         "inherits": "2.0.3",
<         "jsonify": "0.0.0",
<         "resumer": "0.0.0",
<         "through": "2.3.8"
<       },
<       "dependencies": {
<         "deep-equal": {
<           "version": "0.1.2",
<           "resolved": "https://registry.npmjs.org/deep-equal/-/deep-equal-0.1.2.tgz",
<           "integrity": "sha1-skbCuApXCkfBG+HZvRBw7IeLh84="
<         },
<         "defined": {
<           "version": "0.0.0",
<           "resolved": "https://registry.npmjs.org/defined/-/defined-0.0.0.tgz",
<           "integrity": "sha1-817qfXBekzuvE7LwOz+D2SFAOz4="
<         }
<       }
<     },
6503,6507d6175
<     "timespan": {
<       "version": "2.3.0",
<       "resolved": "https://registry.npmjs.org/timespan/-/timespan-2.3.0.tgz",
<       "integrity": "sha1-SQLOBAvRPYRcj1myfp1ZutbzmSk="
<     },
6693,6712d6360
<     "utile": {
<       "version": "0.2.1",
<       "resolved": "https://registry.npmjs.org/utile/-/utile-0.2.1.tgz",
<       "integrity": "sha1-kwyI6ZCY1iIINMNWy9mncFItkNc=",
<       "requires": {
<         "async": "0.2.10",
<         "deep-equal": "1.0.1",
<         "i": "0.3.6",
<         "mkdirp": "0.5.1",
<         "ncp": "0.4.2",
<         "rimraf": "2.6.1"
<       },
<       "dependencies": {
<         "async": {
<           "version": "0.2.10",
<           "resolved": "https://registry.npmjs.org/async/-/async-0.2.10.tgz",
<           "integrity": "sha1-trvgsGdLnXGXCMo43owjfLUmw9E="
<         }
<       }
<     },
7021,7046d6668
<     "winston": {
<       "version": "0.8.3",
<       "resolved": "https://registry.npmjs.org/winston/-/winston-0.8.3.tgz",
<       "integrity": "sha1-ZLar9M0Brcrv1QCTk7HY6L7BnbA=",
<       "requires": {
<         "async": "0.2.10",
<         "colors": "0.6.2",
<         "cycle": "1.0.3",
<         "eyes": "0.1.8",
<         "isstream": "0.1.2",
<         "pkginfo": "0.3.1",
<         "stack-trace": "0.0.10"
<       },
<       "dependencies": {
<         "async": {
<           "version": "0.2.10",
<           "resolved": "https://registry.npmjs.org/async/-/async-0.2.10.tgz",
<           "integrity": "sha1-trvgsGdLnXGXCMo43owjfLUmw9E="
<         },
<         "colors": {
<           "version": "0.6.2",
<           "resolved": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz",
<           "integrity": "sha1-JCP+ZnisDF2uiFLl0OW+CMmXq8w="
<         }
<       }
<     },
7050c6672,6673
<       "integrity": "sha1-o9XabNXAvAAI03I0u68b7WMFkQc="
---
>       "integrity": "sha1-o9XabNXAvAAI03I0u68b7WMFkQc=",
>       "dev": true
```

**Comment by sulkaharo** [comment]
Fixed in latest dev by #3670
